### PR TITLE
Update Dockerfiles 

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: .ci-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build_root_image:
```
Comment (nishant-parekh) [inline .ci-operator.yaml:1]: For ART managed content like `cloud-event-proxy` or `ptp-operator` this update is made through the ART bot.. which would not be the base with `BMER` or `hw-event-proxy` which are manged through CPaaS so we would have to update these manually

**Review by nishant-parekh — COMMENTED** [review]
Just added a comment

**Review by nishant-parekh — APPROVED** [review]

File: .ci-operator.yaml (hunk 1 above)
Comment (jzding) [reply .ci-operator.yaml]: ok, removed .ci-operator.yaml

**Pull request merged**